content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,78 @@
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](/faq/traffic-management/#controlling-inbound-ports) by default.
+- **Improved** Pilot by reducing CPU utilization. The reduction approaches 90% depending on the specific deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
+
+- **Added** trust domain validation for services using mutual TLS. By default, the server only authenticates the requests from the same trust domain.
+- **Added** [labels](/docs/concepts/security/#how-citadel-determines-whether-to-create-service-account-secrets) to control service account secret generation by namespace.
+- **Added** SDS support to deliver the private key and certificates to each Istio control plane service.
+- **Added** support for [introspection](/docs/ops/troubleshooting/controlz/) to Citadel.
+- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor the SDS service.
+- **Added** diagnostics to the Citadel Agent using the `/debug/sds/workload` and `/debug/sds/gateway` on port 8080.
+- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
+- **Improved** SDS security by enforcing the usage of [Kubernetes Trustworthy JWTs](/blog/2019/trustworthy-jwt-sds).
+- **Improved** Citadel Agent logs by unifying the logging pattern.
+- **Removed** support for Istio SDS when using [Kubernetes versions earlier than 1.13](/blog/2019/trustworthy-jwt-sds).
+- **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration in a future release.
+- **Enabled** the Envoy JWT filter by default to improv
+
+## Telemetry
+
+- **Added** Access Log Service [ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v2/service/accesslog/v2/als.proto#grpc-access-log-service-als) support for Envoy gRPC.
+- **Added** a Grafana dashboard for Citadel monitoring.
+- **Added** [metrics](/docs/reference/commands/sidecar-injector/#metrics) for monitoring the sidecar injector webhook.
+- **Added** control plane metrics to monitor Istio's configuration state.
+- **Added** telemetry reporting for traffic destined to the `Passthrough` and `BlackHole` clusters.
+- **Added** alpha support for in-proxy generation of service metrics using Prometheus.
+- **Added** alpha support for environmental metadata in Envoy node metadata.
+- **Added** alpha support for Proxy Metadata Exchange.
```

Comment:
```suggestion
- **Added** alpha support for proxy metadata exchange. This feature enables Envoy proxies to directly exchange workload instance metadata (labels, namespaces, etc.) about themselves. Metadata exchange is considered an alpha feature for this release.
```

Reply:
@louiscryan ^^^ ?

Reply:
too much information.  Please place this content in the core documentation - anywhere will do, and we can link to that from the release notes.

Reply:
same as above comment.

Reply:
I would love to see a list of what each feature does in the feature status pages...  A landing page for all feature status pages in order.  I've spoken with @adammil2000 about this, and believe it would be a valuable pursuit.

Reply:
Sure. But that's not happening for today, right?  Our previous feature status page was links to design docs.

Reply:
that is not happening today, that is correct.